tutorials/google-workspace-mdm-setup-guide.mdx

```diff
@@ -0,0 +1,68 @@
+---
+updated_at: October 02, 2025
+title: Deploy EAP-TLS Wi-Fi certificates to ChromeOS devices with Smallstep and Google Workspace
+html_title: Deploy EAP-TLS to Wi-Fi certificates
+description: Configure EAP-TLS Wi-Fi certificates for Chromebooks using . Complete guide for iOS and macOS wireless security deployment at scale.
+---
+
+Smallstep can integrate with Google Workspace to exchange a SCEP challenge. The SCEP challenge is a shared secret that's used by devices to get a client certificate from Smallstep for a certificate-based Wi-Fi network or VPN server.
```

Comment:
This likely shouldn't be published with the SCEP setup. While we do have a working setup, it's not something we actively support today. So unless users ask for SCEP specifically, I think this can be omitted. Instead, we should have docs on our ChromeOS extension setup with full ACME DA support.